aragog/__init__.py

```diff
@@ -18,7 +18,7 @@
 
 from __future__ import annotations
 
-__version__: str = "0.1.4-alpha"
+__version__: str = "0.1.5-alpha"
```

Comment:
Could we include my PR on logging in the same Github release?

Reply:
Yeah sure!